docs/block-api.md

```diff
@@ -127,36 +127,52 @@ useOnce: true,
 
 * **Type:** `Object`
 
-Optional block extended support features. The following options are supported, and should be specified as a boolean `true` or `false` value:
+Optional block extended support features. The following options are supported, and should be specified as a boolean `true` or `false` value in most of the cases:
```

Comment:
Not sure how to better phrase this exception for an array :)

Reply:
Maybe avoid the second part of the sentence, just leaving:

>Optional block extended support features. The following options are supported:

Reply:
😃